Added SlimeTinker (#145)
Slimefun/Wiki: pages/Addons.md

EMC2 | Seggan | <details><summary>expand</summary>This addon allows for transmutation and mass storage of items by converting them into "Quark-Gluon Plamsa"</details> | N/A | [GitHub](https://github.com/Seggan/EMC2) | [Builds](https://thebusybiscuit.github.io/builds/Seggan/EMC2/master/)
Simple Storage | Sefiraat | <details><summary>expand</summary>This addon adds 3 simple chests (Iron [2 x 45 slot], Gold [4 x 45 slot] and Diamond [6 x 45 slot] but thats just to start! Convert your Diamond chests into Inventory Cells and hook them up to a Network Chest to really get rolling! The Network Chest can access all nearby cells giving a max possible 305,100 stacks of storage for all your items (if you're crazy enough!). Cells can be named and given custom icons for true organisation.</details> | [Wiki](https://github.com/Sefiraat/Simple-Storage/wiki) | [GitHub](https://github.com/Sefiraat/Simple-Storage) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/Simple-Storage/master/)

> SlimeTinker | Sefiraat | <details><summary>expand</summary>SlimeTinker is a Slimefun4 addon that tries it's best to recreate the amazing Tinker's Construct Mod (link) in Vanilla Minecraft. Melt metals, alloy them together, cast metals out into blocks, nuggets, tool parts and more. Build custom tools with properties dependant on the metals used. Tools do not break but instead can be reapired with kits. Swap oput parts as wanted and add special modifiers.</details> | [Wiki](https://github.com/Sefiraat/SlimeTinker/wiki) | [GitHub](https://github.com/Sefiraat/slimeTinker) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/SlimeTinker/master/)

Reviewer: ```suggestion
SlimeTinker | Sefiraat | <details><summary>expand</summary>SlimeTinker is a Slimefun4 addon that tries its best to recreate the amazing [Tinker's Construct Mod](https://www.curseforge.com/minecraft/mc-mods/tinkers-construct) in Vanilla Minecraft. Melt metals, alloy them together, cast metals out into blocks, nuggets, tool parts and more. Build custom tools with properties dependant on the metals used. Tools do not break but instead they can be repaired with kits. Swap out parts as wanted and add special modifiers.</details> | [Wiki](https://github.com/Sefiraat/SlimeTinker/wiki) | [GitHub](https://github.com/Sefiraat/slimeTinker) | [Builds](https://thebusybiscuit.github.io/builds/Sefiraat/SlimeTinker/master/)
```

Reviewer: also the builds link is a 404

Author: Done and done ( the 404 was awaiting build review, in now)
Thanks :)
